### PR TITLE
the imports are shifted to the top

### PR DESCRIPTION
**Pull request by ranbir7**

### CODE REVIEW

File: autoprof/__init__.py [hunk 1]
```diff
@@ -4,15 +4,15 @@
 from . import utils
 from . import fit
 from . import AP_config
+import sys
+import argparse
```
Comment (ConnorStoneAstro) [inline autoprof/__init__.py:4]: That is a good idea. however, import sys and argparse should be the highest level import. Going forward I would like to try and match the PEP 8 rules for imports. See also issue #23

**Comment by ConnorStoneAstro** [comment]
The tests are now failing. I believe the imports have to be of the form `from . import models`. sys and argparse are at the top now though, which is good!

**Comment by ranbir7** [comment]
Tell me once u find a way to solve this
My mind is <strong>stuck</strong> up right now.

**Comment by ConnorStoneAstro** [comment]
Looks like the import `utilsfit` needs to be broken up into `utils, fit` then it should work.

**Review by ConnorStoneAstro — APPROVED** [review]
Everything looks good, and now we follow the import conventions!

**Comment by ranbir7** [comment]
So like every py files will be following this convention?

**Comment by ConnorStoneAstro** [comment]
Yes, going forward we will try to keep to proper import conventions so that it is easier to read :)

**Pull request merged**